src/rpc_server.rs

```diff
@@ -310,21 +360,13 @@ impl AtlasPriorityFeeEstimator {
     ) -> Self {
         let server = AtlasPriorityFeeEstimator {
             priority_fee_tracker,
-            rpc_client: RpcClient::new(rpc_url),
+            rpc_client: Some(RpcClient::new(rpc_url)),
```

Comment:
Adding option for RPC client to make it easier to write tests. The client still gets initialized on regular operations